.gitlab-ci.yml

```diff
@@ -23,16 +23,17 @@ before_script:
   - git clone --branch=master --depth=1 https://github.com/Ensembl/ensembl-variation.git
   - git clone --branch=master --depth=1 https://github.com/Ensembl/ensembl-compara.git
   - git clone --branch=release-1-6-924 --depth=1 https://github.com/bioperl/bioperl-live.git
-  - cpanm -v --installdeps --notest .
+  - cpanm -v --installdeps --notest . --with-all-features
   - ( cd ensembl-test && cpanm -v --installdeps --notest . )
 #  - ( cd ensembl-io && cpanm -v --installdeps --notest . )
 #  - ( cd ensembl-variation && cpanm -v --installdeps --notest . )
   - ( cd ensembl-compara && cpanm -v --installdeps --notest . )
-  - ( cd misc-scripts/xref_mapping && cpanm -v --installdeps --notest . )
   - cpanm -n Devel::Cover::Report::Coveralls
   - cpanm -n DBD::SQLite
   - cp travisci/MultiTestDB.conf.gitlabci.mysql  modules/t/MultiTestDB.conf.mysql
   - cp travisci/MultiTestDB.conf.travisci.SQLite modules/t/MultiTestDB.conf.SQLite
+  - cp travisci/testdb.conf.gitlabci.mysql  testdb.conf.mysql
+  - cp travisci/testdb.conf.travisci.SQLite testdb.conf.SQLite
```

Comment:
This would replace master branch's .gitlab-ci.yml with an older version

Reply:
Rebased against current master again to incorporate changes recently made there.